ffcli/command.go

```diff
@@ -231,12 +231,25 @@ func DefaultUsageFunc(c *Command) string {
 				space = "="
 			}
 
+			// If the help text contains backticks,
+			// e.g. "foo `bar` baz"`, we'll get:
+			//
+			//   argname = "bar"
+			//   usage   = "foo bar baz"
+			//
+			// Otherwise, it's an educated guess for a placeholder,
+			// or an empty string if one couldn't be determined.
+			argname, usage := flag.UnquoteUsage(f)
```

Comment:
This appears to be the major change, is there some prior art that uses `UnquoteUsage` in this way?

Reply:
Apparently flag.FlagSet lol 😮‍💨